_posts/en/newsletters/2021-05-19-newsletter.md

```diff
@@ -0,0 +1,157 @@
+[Bitcoin Improvement Proposals (BIPs)][bips repo], and [Lightning
+BOLTs][bolts repo].*
+
+
+- [Bitcoin Core #21462][] adds tooling for attesting to outputs of Guix builds
+  and verifying these attestations against those of others. After this change,
+  Windows and macOS code signing remain the only missing piece before Guix
+  builds reach feature-parity with Gitian builds.
+
+- [Bitcoin Core GUI #280][] prevents displaying invalid Bitcoin
+  addresses in an error dialog, eliminating the ability to display an
+  arbitrary message in an official-looking dialog.  A simple "invalid
+  address" error is now displayed instead.  (See the PR for illustrative
+  before and after screenshots.)
+
+- [Bitcoin Core #21359][] updates the `fundrawtransaction`, `send` and
+  `walletcreatefundedpsbt` RPCs with a new `include_unsafe` parameter
+  that can be used to spend unconfirmed UTXOs created by other users in
+  the transaction.  This allows fee bumping a transaction using
+  [CPFP][topic cpfp] and was added for that reason by a developer
+  working on implementing [anchor outputs][topic anchor outputs] in the
+  Eclair LN node.  The option should only be used when necessary, as
+  unconfirmed transactions created by other users can be replaced, which
+  may prevent any child transactions from being confirmed.
+
+- [LND #5291][] improves the way LND ensures [PSBTs][topic psbt] for
+  funding transactions only spend segwit UTXOs---segwit UTXOs being
+  required in order to prevent txid malleability from making refund
+  transactions unspendable.  LND previously checked this by looking for
+  the `WitnessUtxo` field in the PSBT, but this field is technically
+  optional for segwit UTXOs and so some PSBT creators don't provide it.
+  The updated code will use the provided value if present or, if it's
+  not present, scan the UTXO set for the necessary information.
+
+- [LND #5274][] limits the maximum amount of funds the node reserves to
+  allow [CPFP][topic cpfp] fee bumping for [anchor outputs][topic anchor
+  outputs] to ten times the per-channel amount.  For nodes with large
+  numbers of channels, this limits their capital requirements.  If they
+  need to close more than 10 channels, they can use the funds received
+  from closing one channel to close the next channel in a domino effect.
+
+- [LND #5256][] allows reading the wallet passphrase from a file.  This
+  is mainly meant for container-based setups where the passphrase is
+  already stored in a file, so using that file directly doesn't create
+  any additional security problems.
+
+- [LND #5253][] adds support for [Atomic Multipath Payment (AMP)][topic
+  multipath payments] invoices across high-level LND RPC commands such as
+  `SendPayment`, `AddInvoice`, and `SubscribeInvoice`. AMP invoices are
+  currently an LND-only feature and only accept HTLCs that have the AMP
+  feature bits sets as well as an AMP payload. This extends [prior
```

Comment:
s/sets/set/?

Reply:
Conner wrote "Enforce AMP feature bits when receiving HTLCs, e.g. AMP invoices should only accept AMP HTLCs, and legacy/MPP HTLCs should only accept non-AMP HTLCs." in the summary, but I can follow up with him to check whether it should be singular.

Reply:
Oh yes, indeed. Fixing.